test/scripts/imgtestlib.py

```diff
@@ -320,3 +320,14 @@ def get_osbuild_commit(distro_version):
         data = json.load(schutzfile)
 
     return data.get(distro_version, {}).get("dependencies", {}).get("osbuild", {}).get("commit", None)
+
+
+def rng_seed_env():
+    """
+    Read the rng seed from the Schutzfile and return it as a map to use as an environment variable with the appropriate
+    key.
+    """
+
+    with open(SCHUTZFILE, encoding="utf-8") as schutzfile:
+        data = json.load(schutzfile)
+    return {"OSBUILD_RNG_SEED": int(data["rngseed"])}
```

Comment:
You make a good point but now I'm thinking whether it *should* be an error for the key not to be set.  If we accidentally remove or rename the key from the Schutzfile, our tests would behave differently from how we expect them to (random values in manifests) and maybe it would be better if we caught that.

I'm going to make it an explicit error instead.

Reply:
An explicit error sounds good to me.